bin/crew

```diff
@@ -725,7 +762,7 @@ def unpack (meta)
       system "ar -p #{meta[:filename]} data.tar.xz | xz -dc#{@verbose} | tar x#{@verbose} -C #{@extract_dir}", exception: true
     when /\.AppImage$/i
       puts "Unpacking 'AppImage' archive, this may take a while..."
-      FileUtils.chmod 0755, meta[:filename], verbose: @fileutils_verbose
+      FileUtils.chmod 0o755, meta[:filename], verbose: @fileutils_verbose
```

Comment:
Not sure why the `o` is necessary but whatever...

Reply:
rubocop whines about octal... so I'm just adding it everywhere to shut it up.

Reply:
Gotcha